server/routes/induction/update/affectAbilityToWorkUpdateController.test.ts

```diff
@@ -132,6 +130,62 @@ describe('affectAbilityToWorkUpdateController', () => {
       expect(req.session.affectAbilityToWorkForm).toBeUndefined()
       expect(req.session.inductionDto).toEqual(inductionDto)
     })
+
```

Comment:
Most of the changes to the "update" controller tests will follow a similar pattern.

I've added a new test for the "get" endpoint that asserts that if this is part of an update journey (updateInductionQuestionSet is on the session) then the page flow history is correctly set.

And then I've added a 2nd test for the "submit" (POST) endpoint that asserts that the service is not called to update the induction, and instead the user is redirected to the correct next screen and the DTO of the Induction that we hold in session has been updated correctly